src/main/java/com/school/project/model/entities/AbstractEntity.java

```diff
@@ -0,0 +1,100 @@
+package com.school.project.model.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Objects;
+
+
+
+public abstract class AbstractEntity extends AbstractBaseEntity implements Serializable {
+
+    private String createdDate;
+    private String updatedDate;
+
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date createdAt;
```

Comment:
should be `createdDate`
you are missing hibernate annotations for date field also.